build_full_frontend_standalone.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
```

Comment:
Could we add an option to build_full_frontend instead of replicating its features?

I can help if needed.

Reply:
it might be better, but I chose not to edit the script to avoid any side effects in production

it also keep things a bit simpler that way

Reply:
I'm concerned by the duplication that may cause maintenance issues, keeping both in sync may be harder.